appengine/leftpad/app.js

```diff
@@ -0,0 +1,34 @@
+// Copyright 2016, Google, Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//   http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+//
+'use strict';
+
+var express = require('express');
+var leftPadIOSDK = require('left-pad-io-sdk');
+
+var app = express();
+
+app.get('/', function(req, res) {
+    leftPadIOSDK.leftPad(res.query.str, res.query.len, res.query.ch).then(function(response){
+          console.log(response); // logs the response 
+          res.status(200).send(response);
```

Comment:
too many spaces here. just two spaces for regular indent, then four spaces only for continuation (i.e., wrapped lines)